docs/catalyst-voting-registration-example.md

```diff
@@ -48,28 +48,36 @@ cardano-hw-cli catalyst voting-key-registration-metadata \
 Create raw transaction with `cardano-cli`, if you don't know how to create simple transaction, check https://github.com/vacuumlabs/cardano-hw-cli/blob/develop/docs/transaction-example.md
 ```
 cardano-cli transaction build-raw \
---mary-era \
---tx-in 270eb90adfb4634fb7e7356dab9a36d1d6c6763e03629ead2e64b59f70217c75#0 \
+--alonzo-era \
+--tx-in "270eb90adfb4634fb7e7356dab9a36d1d6c6763e03629ead2e64b59f70217c75#0" \
 --tx-out addr1q9nz9shd0wh6uevtnr5j4epyqxtwx6953wegv7pfdttr9hzfn5kc55752ehrcspld7ucc0zt8502efdaac4nlajgagasayc3u9+1810000 \
 --fee 190000 \
 --metadata-cbor-file voting_registration.cbor \
---out-file tx.raw
+--out-file tx.raw \
+--cddl-format
 ```
 
-Create payment hwsfile with `cardano-hw-cli`:
+HW wallets expect the transaction CBOR to be in *canonical* format. Unfortunately, cardano-cli sometimes produces incorrectly formatted tx files. Use the following command to fix the formatting issues.
 ```
-cardano-hw-cli address key-gen \
---path 1852H/1815H/0H/0/0 \
---verification-key-file payment.vkey \
---hw-signing-file payment.hwsfile
```

Comment:
I guess I removed this because it doesn't really make sense here - we've already built the tx (with some inputs) and we are only now generating the signing file to sign it? I will at least add a mention that `payment.hwsfile` is the one we used when building the tx.